Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1325,19 +1325,16 @@
       tags:
       - NukeOpsUplink
 
-#- type: listing # DeltaV - Remove syndicats
-#  id: UplinkMobCatMicrobomb
-#  name: uplink-mobcat-microbomb-name
-#  description: uplink-mobcat-microbomb-desc
-#  icon: { sprite: Objects/Devices/communication.rsi, state: old-radio-syndicat }
-#  productEntity: ReinforcementRadioSyndicateSyndiCat
-#  discountCategory: usualDiscounts
-#  discountDownTo:
-#    Telecrystal: 3
-#  cost:
-#    Telecrystal: 6
-#  categories:
-#    - UplinkAllies
+- type: listing # DeltaV - Readd Syndicat with C4
+  id: UplinkMobCatMicrobomb
+  name: uplink-mobcat-microbomb-name
+  description: uplink-mobcat-microbomb-desc
+  icon: { sprite: Objects/Devices/communication.rsi, state: old-radio-syndicat }
+  productEntity: ReinforcementRadioSyndicateSyndiCat
+  cost:
+    Telecrystal: 2
+  categories:
+  - UplinkPointless
```

Comment:
* Add back the old commented out version please.
* Move this to `Resources/Prototypes/_DV/Catalog/Uplink/pointless.yml`
  * give it the ID `id: UplinkMobCat` (so its not conflicting with upstream's)
  * Update the description to `uplink-mobcat-desc` (or whatever ID you ended up giving that locale string)